app/src/components/market/common/set_allowance/index.tsx

```diff
@@ -8,6 +8,7 @@ const Wrapper = styled.div`
   border-radius: 4px;
   border: ${({ theme }) => theme.borders.borderLineDisabled};
   padding: 21px 25px;
+  margin-bottom: 20px;
```

Comment:
Setting the `margin-bottom` of all `SetAllowance` components here causes an unexpectedly large margin in some places, e.g. 
![image](https://user-images.githubusercontent.com/30579067/98580738-548d5a80-2275-11eb-8734-6b6931188cd3.png)

Instead you should be including this in a style prop where this component exists in the component which you want to see the change